metadata/algolia/jobs.xml

```diff
@@ -1,63 +1,9 @@
-                    <parameter name="ExecuteScriptModule.FunctionName">execute</parameter>
-                    <parameter name="ExecuteScriptModule.Transactional">false</parameter>
-                    <parameter name="clearAndRebuild">false</parameter>
-                </parameters>
-            </step>
-            <step step-id="sendProductsDelta" type="ExecuteScriptModule" enforce-restart="false">
-                <description/>
-                <parameters>
-                    <parameter name="ExecuteScriptModule.Module">int_algolia/cartridge/scripts/algolia/job/sendProductsDelta.js</parameter>
-                    <parameter name="ExecuteScriptModule.FunctionName">execute</parameter>
-                    <parameter name="ExecuteScriptModule.Transactional">false</parameter>
-                </parameters>
-            </step>
-        </flow>
-        <rules/>
-        <triggers/>
-    </job>
-
-    <job job-id="AlgoliaCategoriesIndex" priority="0">
-        <description>Export categories to Algolia service for update search index files</description>
-        <parameters/>
-        <flow>
-            <context site-id="RefArch"/>
-            <step step-id="calculateСategoriesDelta" type="ExecuteScriptModule" enforce-restart="false">
-                <description/>
-                <parameters>
-                    <parameter name="ExecuteScriptModule.Module">int_algolia/cartridge/scripts/algolia/job/categoryIndexJob.js</parameter>
-                    <parameter name="ExecuteScriptModule.FunctionName">execute</parameter>
-                    <parameter name="ExecuteScriptModule.Transactional">false</parameter>
-                    <parameter name="clearAndRebuild">false</parameter>
-                </parameters>
-            </step>
-            <step step-id="sendCategoriesDelta" type="ExecuteScriptModule" enforce-restart="false">
-                <description/>
-                <parameters>
-                    <parameter name="ExecuteScriptModule.Module">int_algolia/cartridge/scripts/algolia/job/sendCategoriesDelta.js</parameter>
-                    <parameter name="ExecuteScriptModule.FunctionName">execute</parameter>
-                    <parameter name="ExecuteScriptModule.Transactional">false</parameter>
-                </parameters>
-            </step>
-        </flow>
-        <rules/>
-        <triggers/>
-    </job>
-
-    <job job-id="AlgoliaProductsDeltaExport" priority="0">
-        <description>This is an alternative to the previous delta calculation and sending method which calculates the delta export file by comparing the last full export to the current one, comparing two large XML files.
-This method relies on SFCC's built-in Delta Exports feature to calculate the delta (omitting the delta calculation phase completely and speeding up the process considerably), then enriches and transforms the products before sending them to Algolia for indexing.
+    <job job-id="AlgoliaProductsDeltaIndex_v2" priority="0">
```

Comment:
Didn't we agree on `export` in the end?

Reply:
No we went with Index, I haven't noted all our discussion but it's Algolia terminology and it keeps continuation with the v1 jobs